### PR TITLE
[GC-579] Adds factory extension for seeding based on statics

### PR DESCRIPTION
**Pull request by tom-con**

### CODE REVIEW

**Comment by BriannaGreene** [comment]
lgtm 👍

**Comment by bearfact** [comment]
:+1:

File: rosie/index.js [hunk 1]
```diff
@@ -32,7 +32,25 @@ const extendFactory = (factory, Model) => {
...
 	};
 
-	Object.assign(factory, { create, createMany, clearAll, requestPayload });
+	const seed = async (seedName, params) => {
+		const { seeds } = factory.statics;
+		if (seedName === 'all') {
+			return Promise.all(Object.values(seeds).map(seedDefinition => create({ seedDefinition, ...params })));
+		}
+		const seedDefinition = seeds[seedName];
+		return create({ seedDefinition, ...params });
```
Comment (bearfact) [inline rosie/index.js]: I think this will behave funny if seedDefinition is undefined

Comment (bearfact) [reply rosie/index.js]: Doesn't seedDefinition neeed to default to {} and create needs to be { ...seedDefinition, ...params }

Comment (tom-con) [reply rosie/index.js]: yeah also now I'm seeing I messed up and didn't destructure it

Comment (tom-con) [reply rosie/index.js]: jinx

**Comment by bearfact** [comment]
:+1:

**Comment by tom-con** [comment]
👍

**Pull request merged**